pkg/cmd/cli/cmd/newapp.go

```diff
@@ -502,6 +502,9 @@ func CompleteAppConfig(config *newcmd.AppConfig, f *clientcmd.Factory, c *cobra.
 	if config.ClientMapper == nil {
 		config.ClientMapper = resource.ClientMapperFunc(f.ClientForMapping)
 	}
+	if config.CategoryExpander == nil {
+		config.CategoryExpander = f.CategoryExpander()
+	}
```

Comment:
not sure what this is but if it's needed for the resourcebuilder now, it seems reasonable enough to me.